rules/misc_rules.build_defs

```diff
@@ -457,6 +457,7 @@ def remote_file(name:str, url:str|list, hashes:list=None, out:str=None, binary:b
     if bool(username) != bool(password_file):
         fail(f"Both or neither of username and password_file should be specified")
 
+    lables = labels + ["remote_file"]
```

Comment:
lables -> labels

The perils of dynamic languages!

Reply:
Also WYH: could you change this to a single statement please? i.e.
```
labels += [
    ...
]
```
mildly nicer than repeating the initial addition each time!